packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -15,238 +15,236 @@ import PlaceholderComponent from '../Components/placeholder-component.jsx';
-                            </span>
-                        </div>
+                </div>
+                <div className='statement__account-statistics-item statement__account-statistics-total-withdrawal'>
+                    <div className='statement__account-statistics--is-rectangle'>
+                        <span className='statement__account-statistics-title'>
+                            {localize('Total withdrawals')} {`(${currency})`}
+                        </span>
+                        <span className='statement__account-statistics-amount'>
+                            <Money amount={account_statistics.total_withdrawals} currency={currency} />
+                        </span>
                     </div>
-                    <div className='statement__account-statistics-item'>
-                        <div className='statement__account-statistics--is-rectangle'>
-                            <span className='statement__account-statistics-title'>
-                                {localize('Net deposits')} {`(${currency})`}
-                            </span>
-                            <span className='statement__account-statistics-amount'>
-                                <Money
-                                    amount={account_statistics.total_deposits - account_statistics.total_withdrawals}
-                                    currency={currency}
-                                />
-                            </span>
-                        </div>
+                </div>
+                <div className='statement__account-statistics-item'>
+                    <div className='statement__account-statistics--is-rectangle'>
+                        <span className='statement__account-statistics-title'>
+                            {localize('Net deposits')} {`(${currency})`}
+                        </span>
+                        <span className='statement__account-statistics-amount'>
+                            <Money
+                                amount={account_statistics.total_deposits - account_statistics.total_withdrawals}
+                                currency={currency}
+                            />
+                        </span>
                     </div>
                 </div>
-            </React.Fragment>
-        );
-    }
+            </div>
+        </React.Fragment>
+    );
 
-    render() {
-        const {
-            component_icon,
-            currency,
-            data,
-            date_from,
-            date_to,
-            is_empty,
-            is_loading,
-            is_mx_mlt,
-            is_switching,
-            error,
-            filtered_date_range,
-            handleScroll,
-            handleDateChange,
-            has_selected_date,
-        } = this.props;
+    if (error) return <p>{error}</p>;
 
-        if (error) return <p>{error}</p>;
+    const filter_component = (
```

Comment:
```suggestion
    const FilterComponent = (
```

Reply:
It's not a component. It's a variable holding some `jsx` so per our convention for naming variables, let's continue with this.